External/FEXCore/Source/Interface/Core/GdbServer.cpp

```diff
@@ -611,6 +613,21 @@ GdbServer::HandledPacketType GdbServer::handleXfer(const std::string &packet) {
       return {encode(OSDataString), HandledPacketType::TYPE_ACK};
     }
 
+    if (object == "auxv") {
+      auto CodeLoader = CTX->SyscallHandler->GetCodeLoader();
+      uint64_t auxv_ptr, auxv_size;
+      CodeLoader->GetAuxv(auxv_ptr, auxv_size);
+      std::string data;
+      data.resize(auxv_size * 16);
+
+      for (size_t i = 0; i < auxv_size; ++i) {
+        uint64_t *auxv = reinterpret_cast<uint64_t*>(auxv_ptr + i * 16);
```

Comment:
Looks funky. Could you add a copy explaining what data is being copied here and where the magic value 16 comes from?

Reply:
Changed how this is handled and also fixed an issue where 32-bit wasn't passing auxv correctly.